woocommerce/woocommerce
# fix: dark theme support for "change address" cart button

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The color of the "Change address" button is not always legible with dark-colored background.
This is its current behavior:
<img width="447" alt="Screenshot 2024-10-03 at 11 53 13 AM" src="https://github.com/user-attachments/assets/3ba87844-28b9-4567-841d-094d28da2803">

Changing, so that its color is inherited from the text around it (similar to other buttons on the cart page, like the "Remove item" button):
<img width="444" alt="Screenshot 2024-10-03 at 11 54 00 AM" src="https://github.com/user-attachments/assets/7b33515a-af43-4aa4-bc06-2de47dc096d0">
<img width="479" alt="Screenshot 2024-10-03 at 11 53 09 AM" src="https://github.com/user-attachments/assets/b987019c-c085-45eb-aba6-39707af10ecb">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

- Install a dark-colored theme
  - I used the TT4 theme
  - Go to wp-admin > Appearance > Editor
  - Click on "Styles", select a dark pallette (on the bottom of the page), save
<img width="342" alt="Screenshot 2024-10-05 at 10 10 00 AM" src="https://github.com/user-attachments/assets/df6ac679-4b02-428e-9243-0c98cc85ab8e">
- Ensure you have the block-based cart page
- Add a physical product to cart (e.g.: the "Beanie" default product)
- Notice the change of color on the "Change address" button



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message  fix: "Change address" cart button compatibility w/ dark themes

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
